src/Uno.UI.Tests/Windows_Globalization/Given_CalendarFormatter.cs

```diff
@@ -1,32 +1,29 @@
-﻿using System.Globalization;
+using System.Globalization;
 using System.Linq;
 using Windows.Globalization.DateTimeFormatting;
 using FluentAssertions;
 using FluentAssertions.Execution;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
 
 namespace Uno.UI.Tests.Windows_Globalization
 {
 	[TestClass]
 	public class Given_CalendarFormatter
 	{
 		[TestMethod]
-		[DataRow("day month year", "en-US", "{month.numeric}/{day.integer}/{year.full}")]
-		[DataRow("day month year", "en-CA", "{year.full}-{month.numeric}-{day.integer(2)}")]
-		[DataRow("day month year", "en-GB", "{day.integer(2)}/{month.numeric}/{year.full}")]
-		[DataRow("day month year", "fr-CA", "{year.full}-{month.numeric}-{day.integer(2)}")]
-		[DataRow("day month year", "fr-FR", "{day.integer(2)}/{month.numeric}/{year.full}")]
-		[DataRow("day month year", "hu-HU", "{year.full}. {month.numeric}. {day.integer(2)}.")]
+		[DataRow("day month year", "en-US", "{month.integer}/{day.integer}/{year.full}")]
```

Comment:
NOTE: `month.numeric` that was in the original test expectation is **wrong**. It's not valid by the **pattern grammar**. If you do `new DateTimeFormatter("{month.numeric}/{day.integer}/{year.full}")` on WinUI, that will crash.